jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/startup/CodeWhispererProjectStartupActivity.kt

```diff
@@ -38,8 +37,6 @@ class CodeWhispererProjectStartupActivity : StartupActivity.DumbAware {
         if (!isCodeWhispererEnabled(project)) return
         if (runOnce) return
 
-        promptReAuth(project)
-
```

Comment:
I thought it can still help by doing reAuth once automatically on IDE start? Why do we no longer want this?

Reply:
If we explictly say to grab users attention to re-login when they trigger, then we can resolve this comment.

Reply:
yes the decision was only to show the popup when they trigger the service, either manual or first-time auto-trigger